src/MusicalScore/Graphical/VexFlow/VexFlowMusicSheetCalculator.ts

```diff
@@ -1002,22 +993,22 @@ export class VexFlowMusicSheetCalculator extends MusicSheetCalculator {
     const startX: number = startStaffEntry.PositionAndShape.AbsolutePosition.x - startXOffset;
     const stopX: number = endStaffEntry.PositionAndShape.AbsolutePosition.x + endXOffset;
     vfOctaveShift.PositionAndShape.Size.width = startX - stopX;
-    const textBracket: Vex.Flow.TextBracket = vfOctaveShift.getTextBracket();
-    const fontSize: number = (textBracket as any).font.size / 10;
+    const textBracket: TextBracket = vfOctaveShift.getTextBracket();
+    const fontSize: number = textBracket.fontSizeInPoints / 10;
```

Comment:
@sschmidTU fontSizeInPoints or fontSizeInPixels?

Reply:
pixels. One unit in OSMD is 10 pixels in Vexflow. Not sure how points are handled in Vexflow.

Reply:
If a point in Vexflow is 10 pixels for the default font and settings, maybe that's ok.

Reply:
done